### ISSUE TITLE
New version for new UI

### ISSUE THREAD

**Issue by wayde-ph** [andrewzwicky/PUBGIS#43]
Current UI that is running on the test server no longer works with the current version.

**Comment by andrewzwicky** [comment]
Yep, I'll be working on updating this to include the new map and account for the new UI

**Comment by fergyfresh** [comment]
Does it not work on the minimap or the new map? or both

**Comment by fergyfresh** [comment]
Willing and able to fix the minimap and new map issues, however, I cannot get it to work with my setup. Currently using Windows Subsystem for Linux. Any hints on setup? Do you use python3-dev and virtualenv for development?

**Comment by fergyfresh** [comment]
I'm getting `python setup.py egg_info` has failed.

Are we using `easy_install`?
https://stackoverflow.com/questions/35991403/python-pip-install-gives-command-python-setup-py-egg-info-failed-with-error-c

**Comment by andrewzwicky** [comment]
Not sure what your issue is, but here are some thoughts:

* You can use `pip install pubgis`, which should install PUBGIS along with all the dependencies.  It's personal preference for you on using a virtualenv, but it makes sense to make sure you don't have incorrect dependencies.
* PUBGIS doesn't support linux, I've only tested and developed on Windows proper.  I don't know what Windows Subsystem for Linux is, so I thought I'd mention that.
* AppVeyor shows an installation workflow for Windows [here](https://ci.appveyor.com/project/andrewzwicky/pubgis/build/178)

Appreciate the help!

**Comment by fergyfresh** [comment]
AHHHH, Its windows. Makes sense. I was installing it on Linux.
